packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import uniqueid from 'lodash.uniqueid';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import styles from './Switch.module.scss';
+
+const cx = classNames.bind(styles);
+
+const SWITCH_STATE = Object.freeze({
+  ON: 'On',
+  OFF: 'Off',
+});
+
+const propTypes = {
+  /**
+    * Whether or not the Switch is enabled ("ON").
+    */
+  isOn: PropTypes.bool,
+
+  /**
+    * If true, the user won't be able to toggle the switch.
+    */
+  isDisabled: PropTypes.bool,
```

Comment:
We're inconsistent with this prop name. The majority of form elements use `disabled`, but there are many uses of `isDisabled` throughout other components. It might be worth having a brief conversation to try and settle on a consistent convention.

There is a brief outline of the expectations on our conventions page:

https://engineering.cerner.com/terra-ui/contributing/terra-ui/conventions#for-boolean-props-define-defaultprop-value-as-false

https://engineering.cerner.com/terra-ui/contributing/terra-ui/conventions#exceptions

Reply:
I'd also recommend keeping a consistent prop description format. Using the isDisabled description as an example.

> Whether or not the Switch is enabled ("ON")
> Whether or not the Switch is disabled

Reply:
updated documentation and prop name : [here](https://github.com/cerner/terra-core/blob/895c4fad7bb180e27e1919296ac90eeaa805dea1/packages/terra-switch/src/Switch.jsx#L24-L31)